www/install/step_upgrade/step5.php

```diff
@@ -49,6 +49,12 @@
 $stat = $res->fetch();
 $template = getTemplate('templates');
 
+/* If CEIP is disabled and if is major version of Centreon ask again */
+$aVersion = explode ('.', $_SESSION['CURRENT_VERSION']);
+if (($stat === false || (int)$stat['value'] != 1) && (int)$aVersion[2] == 0) {
```

Comment:
- do not check if $stat === false because it is useless (you set $stat = false on the next line)
- if you cast $aVersion[2] to int, use ===    :  (int)$aVersion[2] === 0